pulpsolv.py

```diff
@@ -377,4 +447,8 @@ def load_repo_units(plugin_manager, repo_name, factory):
             else:
                 print("  - %s" % p)
 
+    print ('---')
+    print('Alternatives:')
+    for alternative in solver.all_alternatives():
+        print({}.format(alternative))
```

Comment:
isn't here syntax error?

Reply:
lol

Reply:
not really syntax but there were no alternatives to manifest